base/range.jl

```diff
@@ -50,13 +50,16 @@ end
     range(start, stop; length, step)
     range(start; length, stop, step)
     range(;start, length, stop, step)
+    range(stop)
 
 Construct a specialized array with evenly spaced elements and optimized storage (an [`AbstractRange`](@ref)) from the arguments.
 Mathematically a range is uniquely determined by any three of `start`, `step`, `stop` and `length`.
 Valid invocations of range are:
 * Call `range` with any three of `start`, `step`, `stop`, `length`.
 * Call `range` with two of `start`, `stop`, `length`. In this case `step` will be assumed
 to be one. If both arguments are Integers, a [`UnitRange`](@ref) will be returned.
+* Call `range` with `step` and either `stop` or `length`. `start` will be assumed to be one.
+* Call `range` with one of `stop` or `length`. `start` and `step` will be assumed to be one.
```

Comment:
Is this accurate and succinct?

```suggestion
* Call `range` with any three of `start`, `step`, `stop`, `length`. The omitted parameter will be computed.
* Call `range` with fewer than three parameters, but including `stop` and/or `length`. If not provided, `start` and/or `step` are assumed to be one.
```